.zuul.yaml

```diff
@@ -1,6 +1,27 @@
 ---
+- job:
+    name: stf-functional-tests-on-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
+      Run the stf functional tests on osp18
+    required-projects:
```

Comment:
If you want to re-use this job in another repo, then the required projects should be updated to include this repo

here is an example:  
https://github.com/infrawatch/service-telemetry-operator/blob/853e4574b5d3c93faa2b9001f8bf18f0a9eddc9e/.zuul.yaml#L90
The stf jobs would not run on the other infrawatch repos until I added STO to the required-projects.